library/Icingadb/Widget/ItemList/ServiceListItemDetailed.php

```diff
@@ -49,6 +49,16 @@ protected function assembleFooter(BaseHtmlElement $footer)
             );
         }
 
+        if ($this->item->state->is_flapping) {
+            $statusIcons->addHtml(new Icon(
+                'random',
+                [
+                    'class' => 'flapping-icon',
+                    'title' => sprintf(t('Service "%s" is in flapping state'), $this->item->display_name)
```

Comment:
Please remove the name. `Service is in flapping state` is sufficient.

Reply:
Not quite.. I guess Yonas added the name because of accessibility. I gave him a task to [work out something in this regard](https://github.com/Icinga/icingadb-web/blob/accessibility-tests/test/TODOS.md) a few weeks ago. Titles should not be ambiguous but should contain the relevant information themselves they're associated with. A screen reader user won't see the visual context with the service and only gets the title read out. Providing info about which host or service this is again, makes it easier.

I'd even go further and suggest to show `Service "<name>" on "<host-name>" is in flapping state`.

Reply:
I see. But what if the name of the service or host is a bit long? The main statement will be at the end of a long sentence.

Reply:
:man_shrugging: We have no layout constraints here.